src/utils/script_args.py

```diff
@@ -70,6 +80,10 @@ def generic_script_init(description: str) -> ScriptArgs:
             period=AccountingPeriod(args.start),
             dune_version=args.dune_version,
         ),
+        orderbook=MultiInstanceDBFetcher(
+            [os.environ["PROD_DB_URL"], os.environ["BARN_DB_URL"]]
+        ),
```

Comment:
TODO - These are new environment variables that need to be added to the ENV sample file.